src/components/TagModal.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+	<Modal size="large" @close="onClose">
+		<div class="modal-content">
+			<h2 class="tag-title">
+				{{ t('mail', 'Add tags') }}
+			</h2>
+			<div>
+				<div>
+					<div class="tag-group__bg button"
+						style="background-color: #31CC7C" />
+					<span class="tag-group__label"
+						style="color: #31CC7C">
+						{{
+							t('mail', 'Work')
```

Comment:
you hard-coded the tags. I think we should already load the account's tags from the server on page load. this gives the same result, but it will ease the next steps with tagging.

Reply:
yes, will polish it after i make it work :D